src/palm-react-native-ui-kit/screens/channels/uikit-app/GroupChannelScreen/GroupChannelInput/BottomMenu.tsx

```diff
@@ -89,6 +89,7 @@ const BottomMenu = ({
         data={menuList}
         keyExtractor={(_, index): string => `menuList-${index}`}
         horizontal
+        showsHorizontalScrollIndicator={false}
```

Comment:
why removed?

Reply:
I thought it was bad look to have a horizontal scroll bar in a small view, so I removed it.

Reply:
I think its okay. we can add some padding below to prevent overlap, but don't we need to show that there are more options on the right?

Reply:
I'll talk to Bibi